### DISCUSSION TITLE
Environment-Variable for self-hosted instances

### DISCUSSION THREAD

**Discussion by patschi** [plausible/analytics#539] [Anything Else]
I think it might be a good additional to implement a specific environment variable for self-hosted instances, to be able to do certain things differently. I think you, Plausible team, are using basically the same code 1:1 for your cloud instance and this could help for self-hosted instances.

To give an example:
1. **Handling `CRON_ENABLED`** (issue #400)
https://github.com/plausible/analytics/blob/039480a0d14186f2f3f13da030dd7a7c39f50673/config/config.exs#L102-L109
In this case the `CRON_ENABLED` environment could be default to `true` for self-hosted scenarios.

2. **Disable cronjob `trial_notification_emails`**
For example `trial_notification_emails`: As those emails are not really useful for self-hosted instances, this cronjob could be disabled when the self-hosted env variable is set.
https://github.com/plausible/analytics/blob/039480a0d14186f2f3f13da030dd7a7c39f50673/config/config.exs#L131

3. **Disable cronjob `provision_ssl_certificates`**
https://github.com/plausible/analytics/blob/039480a0d14186f2f3f13da030dd7a7c39f50673/config/config.exs#L127

4. **Branding**
At certain places (emails, footer, etc) there's Plausible.io-related branding. While branding/credits are fine, those could be replaced to the admin's email address once self-hosted instance is in use.

**Comment by ukutaht** [comment]
@patschi Spot on, I've been thinking about doing something like this as well. I will probably add it for the next release
